docker-compose.yml

```diff
@@ -13,23 +13,23 @@ services:
     depends_on:
       - mongodb
       
-  backend:
```

Comment:
I think you were running each component separately for development rather than using a single command. For consistency and to avoid confusion, please update the configurations to match the current main setup